index.html

```diff
@@ -3,8 +3,10 @@
   <head>
     <meta charset="utf-8">
     <title>Mini Weather App</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css" integrity="sha256-HxaKz5E/eBbvhGMNwhWRPrAR9i/lG1JeT4mD6hCQ7s4=" crossorigin="anonymous" />
     <link rel="stylesheet" href="css/style.css">
```

Comment:
Make sure your custom CSS is the last file loaded (bootstrap , normalize, googlefont) should all be first.